modules/ROOT/Edits/edifact-receive-read-settings.adoc

```diff
@@ -0,0 +1,100 @@
+= Configure Receive Settings for Inbound EDIFACT messages
+
+Partner Manager receive settings include the EDIFACT acknowledgments and validation rules to apply when processing an inbound EDIFACT message from the specified partner.
+
+To configure EDIFACT receive settings:
+
+. In the navigation menu, select *Partners*.
+. Select the partner to which these settings apply.
+. In the *Validation and acknowledgment settings* section's *Receive from <partner-name>* column, click *EDIFACT*.
+. Select a link to configure:
+* <<ack-settings,Acknowledgment settings>>
+* <<validation-settings,Validation settings>>
+* <<control-numbers,Control Numbers>>
+. Click *Save*.
+
+[NOTE]
+It can take up to five minutes or so for your configuration changes to sync with Mule.
+
+[[ack-settings]]
+== Acknowledgment Settings
+
+Optionally configure acknowledgment settings for each Inbound EDIFACT message flow. To configure acknowledgement settings:
+
+* Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
```

Comment:
The next three topics are reference topics, each containing a solitary task step, highlighted as a bullet list item.
This seems a bit odd to me.
Maybe we should move the action into the intro, something like this:

```suggestion
== Acknowledgment Settings

Optionally configure acknowledgment settings for each Inbound EDIFACT message flow by enabling the *Send CONTRL functional acknowledgment when messages fail to route to a message flow* setting.
```